src/malloy/runtime.py

```diff
@@ -129,6 +132,10 @@ async def run(self, connection, query=None, named_query=None):
     if sql is None:
       return None
 
+    connections = re.search(r"^-- connection: (.*)", sql)
+    if connections:
+      connection = connections.group(1)
+
```

Comment:
This uses https://github.com/malloydata/malloy-service/pull/4 to dynamically get the connection name, which still can only be "duckdb" or "bigquery" because I'm not exposing a way to add connections yet, but that should be doable.

Reply:
This looks a bit hackier than I would like to see, we can update the proto definitions to handle this information rather than magically parsing strings.